### DISCUSSION TITLE
Dialogues on the Big Bang Debate and a workshop Friday Nov. 3rd

### DISCUSSION THREAD

**Discussion by RedshiftDrift** [a-cosmology-group/acg#180] [General]
Recent dialogues:

1) Dialog on the Big Bang Debate: Dr Rajendra Gupta and Eric Lerner
[https://youtu.be/HWA273rAKLk](https://youtu.be/HWA273rAKLk)

2) Dialogs on the Big Bang Debate: Dr Francesco Sylos-Labini
[https://youtu.be/J88nQ1gbQHo](https://youtu.be/J88nQ1gbQHo)

Workshop Friday morning:

3) Friday, Nov. 3 from 9:15 Am to 2:15 PM EDT: LPPFusion will host an online workshop on the question: “Is Cosmic Expansion Valid in the Epoch of JWST?” For details and Zoom link see [https://youtu.be/J88nQ1gbQHo](https://youtu.be/J88nQ1gbQHo)

  Participants include *Rajendra Gupta*, University of Ottawa; *Eric Lerner*, LPPFusion; *Martin Lopez-Corredoira*, Instituto de Astrofisica de Canarias; *Leandros Perivolaropoulos*, University of Ioannina; *Riccardo Scarpa*, Instituto de Astrofisica de Canarias; *Francesco Sylos-Labini*, Enrico Fermi Research Center and *Alessandro Trinchera*, Eberhard Karls University of Tübingen.

**Comment by gfellow** [comment]
Enjoyed the Lerner interviews and have just joined github specifically for this site, been doing an initial blunder through this site in an attempt to get my bearings, so forgive me if I am not posting in the correct location.
Make a living as an artist, a plasma layman for forty+ years, think a laboratory experiment can prove that an electrical discharge contains a moment of gravitational induction; [gravity without a corresponding quantity of mass](https://youtu.be/PNzaLRozQD4?si=YTSdQ0ENR4ZLmTFK). 
OK, all for now, off to bed.

**Reply by RedshiftDrift** [reply]
@gfellow Welcome to ACG discussions on GitHub.

This post is not relevant to Eric Lerner's Big Bang debate.  Next time start a new conversation on the discussions main page, and use the "Gravitation models" category for topics on gravitational induction.

**Comment by RedshiftDrift** [comment]
The videos from the workshop have been published
https://www.youtube.com/@LPPFusion/videos
